src/main/java/org/dependencytrack/policy/PolicyEngine.java

```diff
@@ -140,7 +141,7 @@ public PolicyViolation.Type determineViolationType(final PolicyCondition.Subject
         }
         return switch (subject) {
             case CWE, SEVERITY, VULNERABILITY_ID -> PolicyViolation.Type.SECURITY;
-            case AGE, COORDINATES, PACKAGE_URL, CPE, SWID_TAGID, COMPONENT_HASH, VERSION, VERSION_DISTANCE ->
+            case AGE, COORDINATES, PACKAGE_URL, CPE, SWID_TAGID, COMPONENT_HASH, VERSION, VERSION_DISTANCE, EPSS ->
```

Comment:
As EPSS is a metric related to vulnerabilities, I think it should map to `PolicyViolation.Type.SECURITY`.